packages/probers/cypress/e2e/repostTrack.cy.ts

```diff
@@ -12,7 +12,7 @@ describe('Repost Track', () => {
     const base64Entropy = Buffer.from(user.entropy).toString('base64')
     cy.visit(`${track.route}?login=${base64Entropy}`)
 
-    cy.findByText(user.name, { timeout: 20000 }).should('exist')
+    cy.findByText(user.name, { timeout: Cypress.env('initialLoadTimeout') }).should('exist')
```

Comment:
Do we even need to specify initalLoadTimeout if that is the default now? Or is this just a new field we've added to prevent magic numbers in the test?

Reply:
yeah we can probably just drop this tbh. I'll do! was trying to understand how custom configs work and learned it has to be in .env

Reply:
On second thought, maybe not -- because some of these were like 20s in the non dev env